apps/angular/projection/src/app/component/city-card/city-card.component.ts

```diff
@@ -1,13 +1,46 @@
+import { AsyncPipe } from '@angular/common';
 import { Component, OnInit } from '@angular/core';
+import { Observable } from 'rxjs';
+import { CityStore } from '../../data-access/city.store';
+import {
+  FakeHttpService,
+  randomCity,
+} from '../../data-access/fake-http.service';
+import { City } from '../../model/city.model';
+import { CardComponent } from '../../ui/card/card.component';
+import { ListItemComponent } from '../../ui/list-item/list-item.component';
 
 @Component({
   selector: 'app-city-card',
-  template: 'TODO City',
+  templateUrl: './city-card.component.html',
+  styles: [
+    `
+      .bg-light-blue {
+        background-color: #9de1dd;
+      }
+    `,
+  ],
   standalone: true,
-  imports: [],
+  imports: [CardComponent, ListItemComponent, AsyncPipe],
 })
 export class CityCardComponent implements OnInit {
-  constructor() {}
+  cities$!: Observable<City[]>;
```

Comment:
you should use signal now to deal with the template

Reply:
Yeah! yet to get grasp of it.